code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -8119,8 +8044,7 @@ private void processAbilityListsOnAdd(CDOMObject cdo,
 		{
 			if (list instanceof AbilityList)
 			{
-				CDOMReference r = ref;
-				processAbilityList(cdo, r);
+				processAbilityList(cdo, (CDOMReference) ref);
```

Comment:
I suspect this won't compile on some compilers which is why it was separated out, so we may want to merge this PR alone